src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,60 @@
+package lotto.domain;
+
+import java.util.*;
+
+public class Lottos {
+
+    public final List<Lotto> lottos;
+
+    public Lottos(final List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public List<Lotto> getLottos() {
+        return Collections.unmodifiableList(lottos);
+    }
+
+    public static Lottos union(final Lottos first, final Lottos second) {
+        return new Lottos(new ArrayList<>() {{
+            addAll(first.lottos);
+            addAll(second.lottos);
+        }});
+    }
+
+    public LottoWinningStatistics giveOutWinningStatistics(final int lottoPurchaseAmount,
+                                                           final WinningLottoNumbers winningLottoNumbers) {
+        Map<LottoWinningType, Integer> lottoWinningTypeCountMap
+                = countLottoWinningTypes(this.lottos, winningLottoNumbers);
```

Comment:
인스턴스 변수는 메서드 인자로 직접 넘겨주지 않아도 될 것 같아요.
```suggestion
                = countLottoWinningTypes(winningLottoNumbers);
```

Reply:
반영하겠습니다~